frontend/public/components/resource-dropdown.tsx

```diff
@@ -43,9 +42,10 @@ const blacklistResources = ImmutableSet([
   'extensions/v1beta1.ReplicationControllerDummy',
 ]);
 
-const DropdownItem: React.SFC<DropdownItemProps> = ({ model, showGroup }) => (
+const DropdownItem: React.SFC<DropdownItemProps> = ({ model, showGroup, checked }) => (
   <>
-    <span className="co-resource-item">
+    <span className={'co-resource-item'}>
+      <Checkbox id={model.apiGroup + model.apiVersion + model.kind} isChecked={checked} />
```

Comment:
I'm not sure we'd see this in practice, but there's an edge case where this isn't unique.

group: foobar
version: baz
kind: qux

and

group: foo
version: barbaz
kind: qux

will give the same ID

I'd use `:` as a separator since it's allowed for `id` and isn't allowed for group/version/kind values

```suggestion
      <Checkbox id={`${model.apiGroup}:${model.apiVersion}:${model.kind}`} isChecked={checked} />
```

Is `id` really required here? I'd be best to just leave that off honestly unless Patternfly requires it.

Reply:
Id is required and it has to be unique.